spec/features/admin/invoice/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'admin invoices index' do
+  before(:each) do
+    @customer1 = Customer.create!(first_name: 'Cowboy', last_name: 'Smith')
+    @customer2 = Customer.create!(first_name: 'Mercy', last_name: 'Champion')
+    @invoice1 = Invoice.create!(status: 0, customer_id: "#{@customer1.id}")
+    @invoice2 = Invoice.create!(status: 0, customer_id: "#{@customer1.id}")
+    @invoice3 = Invoice.create!(status: 0, customer_id: "#{@customer2.id}")
+  end
+
+  it 'shows a list of all Invoice ids in the system' do
+    visit '/admin/invoices'
+    expect(page).to have_content(@invoice1.id)
+    expect(page).to have_content(@invoice2.id)
+    expect(page).to have_content(@invoice3.id)
+  end
+
+  it 'id is a link to the invoice show page' do
+    visit '/admin/invoices'
+    expect(page).to have_link("#{@invoice1.id}")
+    expect(page).to have_link("#{@invoice2.id}")
```

Comment:
should we test that the links are working by adding e`xpect(current_path).to eq(admin_invoices_path(@invoice1))` or something along those lines?